src/openrct2/windows/themes.c

```diff
@@ -245,6 +245,7 @@ static rct_windowclass window_themes_tab_6_classes[] = {
 	WC_MULTIPLAYER,
 	WC_PLAYER,
 	WC_CHAT,
+	WC_VIEW_CLIPPING,
```

Comment:
Put this one below `WC_TILE_INSPECTOR` to match up with the definition in `Themes.cpp`.